--- qcomicbook-0.4.3/data/qcomicbook.desktop	2009-08-12 21:45:47.000000000 +0200
+++ qcomicbook-0.4.3.new/data/qcomicbook.desktop	2009-09-14 23:10:54.000000000 +0200
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=qcomicbook
 GenericName=ComicBook Viewer
 GenericName[pl]=Przeglądarka komiksów
@@ -8,9 +7,9 @@
 Comment[tr]=QComicBook cbz, cbr, cbt biçimindeki çizgi roman arşivlerini görüntülemek için kullanılır. Sayfa ön görünümü, sayfa döşeme, yer imi, mangalar için sayfa sırasını ters gösterme gibi çizgi roman okumayı kolaylaştırıcı özellikleri bulunuyor.
 Comment[pl]=Przeglądarka do archiwów z komiksami
 Exec=qcomicbook
-Icon=qcomicbook.png
+Icon=qcomicbook
 Terminal=false
 Type=Application
 StartupNotify=false
-Categories=Application;Graphics;Viewer;
+Categories=Graphics;Viewer;
 MimeType=application/x-cbz;application/x-cbr;application/x-cbt;
